### DISCUSSION TITLE
accountability example

### DISCUSSION THREAD

**Discussion by dkamali-mc** [directus/directus#15961] [Archive]
Hey everyone, 

I'm looking for guidance on this one, I'm building the frontend site, and my dashboard has a dropdown menu that suppose to show recent user activities similar to the log trail in accounting systems. 

The [documentation here](https://docs.directus.io/configuration/data-model/collections.html#accountability) discusses enabling the feature of tracking changes in a given collection. 

I'm looking for a graphql or API example on how to retrieve recent activities made (type of change, "create, update, delete," author and time).

**Comment by rijkvanzanten** [comment]
That'd be https://docs.directus.io/reference/system/activity.html for the actual activity records you're looking for, and https://docs.directus.io/reference/system/revisions.html for the actual changes that go with those activity records 👍🏻
